src/index.js

```diff
@@ -1,8 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './components/App';
+import { Provider } from 'react-redux';
+import Router from './components/Router/Router';
+
 import './scss/style.scss';
 
-ReactDOM.render(<App />, document.getElementById('app'));
+import store from './store/store';
+
+ReactDOM.render(
+  <Provider store={store}>
+    <Router />
```

Comment:
You dont need to create routes and router folders. Just place code from `Router.js` file here or create some root component. Or even combine folders  `router` and `routes` for keeping router logic in the 1 place.